pkg/runtime/runtime.go

```diff
@@ -730,6 +742,10 @@ func (a *DaprRuntime) initPubSub() error {
 				continue
 			}
 
+			allowedSubscriptions = scopes.GetScopedTopics(scopes.SubscriptionScopes, a.runtimeConfig.ID, properties)
+			a.allowedPublishings = scopes.GetScopedTopics(scopes.SubscriptionScopes, a.runtimeConfig.ID, properties)
```

Comment:
nit: usage of prefix allowed in allowedPublishings was confusing for me when I read the code as from the spec allowed has different meaning, allowedTopis.  Can it be renamed to something like scoped or whitelisted

Reply:
Yep I'll cook something up.

Reply:
Cooked.